src/app/features/location/map/map.component.ts

```diff
@@ -1,93 +1,81 @@
 import {
   AfterViewInit,
   Component,
+  ElementRef,
   EventEmitter,
   Input,
   Output,
+  ViewChild,
 } from "@angular/core";
 import * as L from "leaflet";
-import { Observable, Subject } from "rxjs";
-import { debounceTime, map } from "rxjs/operators";
+import { Observable, Subject, timeInterval } from "rxjs";
+import { debounceTime, filter, map } from "rxjs/operators";
 import { Coordinates } from "../coordinates";
 import { Entity } from "../../../core/entity/model/entity";
-
-const iconRetinaUrl = "assets/marker-icon-2x.png";
-const iconUrl = "assets/marker-icon.png";
-const shadowUrl = "assets/marker-shadow.png";
-const iconDefault = L.icon({
-  iconRetinaUrl,
-  iconUrl,
-  shadowUrl,
-  iconSize: [25, 41],
-  iconAnchor: [12, 41],
-  popupAnchor: [1, -34],
-  tooltipAnchor: [16, -28],
-  shadowSize: [41, 41],
-});
-L.Marker.prototype.options.icon = iconDefault;
+import { getHueForEntity } from "../map-utils";
+import { ConfigService } from "../../../core/config/config.service";
+import { UiConfig } from "../../../core/ui/ui-config";
 
 @Component({
   selector: "app-map",
   templateUrl: "./map.component.html",
   styleUrls: ["./map.component.scss"],
 })
 export class MapComponent<T extends Entity = Entity> implements AfterViewInit {
-  // TODO this should be configurable
   private readonly start_location: L.LatLngTuple = [52.4790412, 13.4319106];
 
-  @Input() set marked(coordinates: Coordinates | Coordinates[]) {
-    if (!coordinates) {
-      return;
-    }
-    if (Array.isArray(coordinates)) {
-      this.setMultipleMarkers(coordinates);
-    } else {
-      this.setMarker(coordinates);
-    }
-    if (this.map) {
-      this.showMarkersOnMap();
-    }
+  @ViewChild("map") private mapElement: ElementRef<HTMLDivElement>;
+
+  @Input() height = "200px";
```

Comment:
I guess getting this just from css properties assigned to the parent container didn't work?